python/pyxet/pyxet/util.py

```diff
@@ -33,11 +43,10 @@ def _should_load_aws_credentials():
     return True
 
 
-def _get_fs_and_path(uri):
+def _get_fs_and_path(uri, strip_trailing_slash = True):
```

Comment:
this param `strip_trailing_slash` doesn't seem to be used anywhere?

Reply:
Very good catch.  It should be passed to _path_normalize.

Reply:
Very good catch.  It should be passed to _path_normalize.